tests/modules/cluster.c

```diff
@@ -46,6 +46,19 @@ int PingallCommand(ValkeyModuleCtx *ctx, ValkeyModuleString **argv, int argc) {
     return ValkeyModule_ReplyWithSimpleString(ctx, "OK");
 }
 
+void DingReceiver(ValkeyModuleCtx *ctx, const char *sender_id, uint8_t type, const unsigned char *payload, uint32_t len) {
+    ValkeyModule_Log(ctx, "notice", "DING (type %d) RECEIVED from %.*s: '%.*s'", type, VALKEYMODULE_NODE_ID_LEN, sender_id, (int)len, payload);
+    ValkeyModule_SendClusterMessage(ctx, NULL, MSGTYPE_DONG, "Message Received!", 17);
+    ValkeyModuleCallReply *reply = ValkeyModule_Call(ctx, "INCR", "c", "dings_received");
```

Comment:
This will be tricky.`INCR` is a write operation so it can only be applied on a primary. For testing purpose, log verification is sufficient. So, we could remove this operation.